zone/lua_mob.cpp

```diff
@@ -3050,6 +3059,55 @@ bool Lua_Mob::HasSpellEffect(int effect_id) {
 	return self->HasSpellEffect(effect_id);
 }
 
+Lua_Mob_List Lua_Mob::GetCloseMobList() {
+	Lua_Safe_Call_Class(Lua_Mob_List);
+
+	Lua_Mob_List ret;
+
+	const auto& l = entity_list.GetCloseMobList(self);
+
```

Comment:
Since we know the container size we can reserve the memory up front instead of continually allocating memory in the loop. It is always faster to bulk allocate.

eg

```cpp
ret.reserve(l.size());
```

Reply:
We can't necessarily do this for cases where we have to calculate distance yet